### PR TITLE
replace MUI CircularProgress

### LINKED ISSUE #4999: Replace Material UI components in `src/Components/Shifting/ShiftDetailsUpdate.tsx`:
```tsx
18: import { CircularProgress } from "@material-ui/core";\n28: } from "@material-ui/core";
```

### Tasks

Ensure all the below tasks are accomplished if applicable to the file being modified in the PR.

- [ ] Migrate all Material UI components to use new Care UI / custom components. (ref: [🛠️ Tailwinding Care](https://github.com/coronasafe/care_fe/issues/3742))
- [ ] Migrate all Material UI icons to use [🏥 **Care Icons**](https://github.com/coronasafe/care_fe/pull/4071).
- [ ] Migrate `PageTitle` to `Page` component if any. (See [example](https://github.com/coronasafe/care_fe/pull/4796/files#diff-8e39f154ccf13c9dedd0f968d21a0abbde450e88df9724a4e7d4264d487a7846))

For more information see the epic issue that tracks this issue.

### PR DESCRIPTION
**Pull request by ayushjnv1**
## Proposed Changes

- Fixes #4999 

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by ayushjnv1** [comment]
fix issue #4999

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Shifting/ShiftDetailsUpdate.tsx
Comment (rithviknishad) [inline src/Components/Shifting/ShiftDetailsUpdate.tsx:1]: @ayushjnv1 this file contains `PageTitle` that can be changed to `Page` wrapper component.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Shifting/ShiftDetailsUpdate.tsx
Comment (ayushjnv1) [reply src/Components/Shifting/ShiftDetailsUpdate.tsx:1]: ok I will do that.

File: src/Components/Shifting/ShiftDetailsUpdate.tsx [hunk 1]
```diff
@@ -245,11 +245,11 @@ export const ShiftDetailsUpdate = (props: patientShiftProps) => {
...
   return (
-    <div className="px-2 pb-2">
-      <PageTitle
-        title={t("update_shift_request")}
-        backUrl={`/shifting/${props.id}`}
-      />
+    <Page
+      title={t("update_shift_request")}
+      backUrl={`/shifting/${props.id}`}
+      className="px-2 pb-2"
```
Comment (rithviknishad) [inline src/Components/Shifting/ShiftDetailsUpdate.tsx]: Avoid using custom padding when using `Page` component unless necessary. Newer components allows us to have consistent designs across the platform out of the box.
```suggestion
```

**Review by rithviknishad — APPROVED** [review]

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@ayushjnv1 we have a slight problem. All the below marked fields are coming from `HelperInputFields` which indirectly uses `MaterialUI` components.

Could you use the new FormField components for all fields in this page?

image

cc: @nihal467

**Comment by ayushjnv1** [comment]
ok I will do that

**Comment by rithviknishad** [comment]
@ayushjnv1 if you could merge the latest develop branch now, you'll be able to see all the old components have been renamed to `Legacy<ComponentName>` so that it's easier to track which needs to be migrated.

Ref: 2c063da4d1f50b008500064f967523c84621cc42

**Comment by rithviknishad** [comment]
Closing as completed via #5415

**Pull request closed without merging**